docs/reference/connector/apis/update-connector-filtering-api.asciidoc

```diff
@@ -30,65 +33,42 @@ Updates the `filtering` configuration of a connector. Learn more about filtering
 [[update-connector-filtering-api-request-body]]
 ==== {api-request-body-title}
 
-`filtering`::
-(Required, array) The filtering configuration for the connector. This configuration determines the set of rules applied for filtering data during syncs.
-
-Each entry in the `filtering` array represents a set of filtering rules for a specific data domain and includes the following attributes:
-
-- `domain` (Required, string) +
-Specifies the data domain to which these filtering rules apply.
-
-- `active` (Required, object) +
-Contains the set of rules that are actively used for sync jobs. The `active` object includes:
-
-  * `rules` (Required, array of objects) +
-  An array of individual filtering rule objects, each with the following sub-attributes:
-    ** `id` (Required, string) +
-    A unique identifier for the rule.
-    ** `policy` (Required, string) +
-    Specifies the policy, such as "include" or "exclude".
-    ** `field` (Required, string) +
-    The field in the document to which this rule applies.
-    ** `rule` (Required, string) +
-    The type of rule, such as "regex", "starts_with", "ends_with", "contains", "equals", "<", ">", etc.
-    ** `value` (Required, string) +
-    The value to be used in conjunction with the rule for matching the contents of the document's field.
-    ** `order` (Required, number) +
-    The order in which the rules are applied. The first rule to match has its policy applied.
-    ** `created_at` (Required, datetime) +
-    The timestamp when the rule was added.
-    ** `updated_at` (Required, datetime) +
-    The timestamp when the rule was last edited.
-
-  * `advanced_snippet` (Required, object) +
-  Used for {enterprise-search-ref}/sync-rules.html#sync-rules-advanced[advanced filtering] at query time, with the following sub-attributes:
-    ** `value` (Required, object) +
-    A JSON object passed directly to the connector for advanced filtering.
-    ** `created_at` (Required, datetime) +
-    The timestamp when this JSON object was created.
-    ** `updated_at` (Required, datetime) +
-    The timestamp when this JSON object was last edited.
-
-  * `validation` (Required, object) +
-  Provides validation status for the rules, including:
-    ** `state` (Required, string) +
-    Indicates the validation state: "edited", "valid", or "invalid".
-    ** `errors` (Required, object) +
-    Contains details about any validation errors, with sub-attributes:
-      *** `ids` (Required, string) +
-      The ID(s) of any rules deemed invalid.
-      *** `messages` (Required, string) +
-      Messages explaining what is invalid about the rules.
-
-- `draft` (Required, object) +
-An object identical in structure to the `active` object, but used for drafting and editing filtering rules before they become active.
+`rules`::
+(Optional, array of objects)
+An array of {enterprise-search-ref}/sync-rules.html#sync-rules-basic[basic sync rules], each with the following sub-attributes:
+- `id` (Required, string) +
```

Comment:
nit: we normally use * for lists in asciidoc, but if it's rendering OK not a big deal